tools/osbuilder/rootfs-builder/cbl-mariner/config.sh

```diff
@@ -5,7 +5,6 @@
 OS_NAME=cbl-mariner
 OS_VERSION=${OS_VERSION:-2.0}
 LIBC="gnu"
-PACKAGES="core-packages-base-image ca-certificates device-mapper"
-PACKAGES="core-packages-base-image ca-certificates"
+PACKAGES="core-packages-base-image ca-certificates device-mapper opa"
```

Comment:
I believe OPA is already part of rootfs. So, if you are adding it here, you should remove it from the rootfs build scripts.

Reply:
I forgot to remove this - it's removed from config.sh now and still in rootfs.sh